.github/workflows/integration_tests_pointer.yml

```diff
@@ -0,0 +1,13 @@
+# Workflow is unable to retrieve data about previous workflows (the same file).
+# This workflow is a way to preserve last run date for full_suite_integration_tests.yml workflow
+# which is available from within that workflow
+name: Integration Tests Pointer
+on:
+  repository_dispatch:
+    types: [ integration-pointer]
+jobs:
+  point:
+    runs-on: ubuntu-latest
+    steps:
+      - run: |
+          echo "Integration tests pointer updated: $(date +'%Y-%m-%d')"
```

Comment:
how is it exactly preserving the date via an echo?

Reply:
Hm... I thought the description is just enough. Maybe I should add additional info...
Long story short, let's say we have workflow described in `a.yml`. During `a` run we want to fetch some data about previous `a` runs (like a time when it was previously finished). We use https://docs.github.com/en/free-pro-team@latest/rest/reference/actions#list-workflow-runs. Due to limitations, we get an empty array as a response.
But we can use the same endpoint to get info about another workflow (so inside `a` workflow we can get info about `b`). Those pointers are "just to be triggered" meaning they do nothing like preserves (by being triggered) timestamp, date etc.
For dependency update workflow looks like:
1. start dependency update workflow
2. proceed with all the jobs/steps but in parallel trigger (with repository dispatch) event "pointer" workflow
3. next time dependency update will be launched it will look for the latest date of "pointer" workflow (because it can not fetch data about itself)

> how is it exactly preserving the date via an echo?

Echo is just for debugging purposes. It could be nothing there actually